README.md

```diff
@@ -2,14 +2,12 @@
 
 [![Build Status](https://github.com/amzn/ion-go/workflows/Go%20Build/badge.svg)](https://github.com/amzn/ion-go/actions?query=workflow%3A%22Go+Build%22)
 
-This package is for parsing and writing text and binary based Ion data.
+Amazon Ion ( http://amzn.github.io/ion-docs/ ) library for Go
 
-***This package is considered experimental, under active early development,  and the API is subject to change.***
+***This package is consider beta. While the API is relatively stable it is still subject to change***
 
-http://amzn.github.io/ion-docs/docs/spec.html
-
-It aims to be efficient by only evaluating values when they are accessed, and retaining the original
-text or binary representation where possible for efficient re-serialization of the same form.
+This package is based on work from David Murray ([fernomac](https://github.com/fernomac/)) on https://github.com/fernomac/ion-go.
+The Ion team greatly appreciates David's contributions to the Ion community.
```

Comment:
😊

(Thanks again for picking this up and running with it, I'm excited to have an officially-supported version!)